packages/webapp/src/App.tsx

```diff
@@ -55,6 +55,12 @@ const App = () => {
     const SentryRoutes = Sentry.withSentryReactRouterV6Routing(Routes);
     const env = useStore((state) => state.cookieValue);
     const signout = useSignout();
+    const setShowGettingStarted = useStore((state) => state.setShowGettingStarted);
+    const showGettingStarted = useStore((state) => state.showGettingStarted);
+
+    useEffect(() => {
```

Comment:
should we use `useState` instead?
```
const [showGettingStarted, setShowGettingStarted] = useState(env === 'dev' && (isCloud() || isLocal()));
```

Reply:
The same condition is reused in an other file, so it's less ideal